### ISSUE TITLE
Support {% highlight language %}

### ISSUE THREAD

**Issue by agrafix** [uranusjr/macdown#428]
It seems that currently the jekyll flavored `{% highlight language %} {% endhighlight %}` is not supported.  Are there plans to support this or am I missing something?

**Comment by uranusjr** [comment]
You’re correct, this syntax is not currently supported. What does it do?

**Comment by agrafix** [comment]
It's the jekyll equivalent (see http://jekyllrb.com/docs/templates/#code-snippet-highlighting ) to the tripple backslash syntax highlighting.

**Comment by uranusjr** [comment]
I see. Does Jekyll support the triple backtick syntax? If it does, I would recommend using it since it is more mainstream anyway.

**Comment by agrafix** [comment]
Yes, you can enable that, but it's not trivial and Github Pages does not support it :-( See https://github.com/jekyll/jekyll/issues/2709 and http://sacha.me/articles/jekyll-rouge/

**Comment by ahwillia** [comment]
+1 for this. I am having trouble setting up a markdown interpreter that has both syntax highlighting for code and is compatible with mathjax.

**Comment by franklinyu** [comment]
You can use triple-tilde syntax:

```markdown
~~~ruby
def two
  1 + 1
end
~~~
```

Which is supported both in Kramdown (default rendering engine of Jekyll) and MacDown.
